providers/statsig/README.md

```diff
@@ -74,4 +74,7 @@ As it is limited, evaluation context based tests are limited.
 See [statsigProviderTest](./src/test/java/dev/openfeature/contrib/providers/statsig/StatsigProviderTest.java)
 for more information.
 
+## Release Notes
 
+### 0.3.0
+ Migrated to Java Core according to [Migration guide](https://docs.statsig.com/server-core/migration-guides/java#java-migration-steps).
```

Comment:
Definitely mention that this is a major breaking change and that there is migration needed.  Also mention that the last version of non-javacore requires a downgrade.